MachineLearningPrototype/utils/preprocess_data.py

```diff
@@ -15,3 +16,41 @@ def extract_quotation_content(content):
     if (len(substring) >= 1):
         result=' '.join(substring)
         return result
+
+def query_trss_for_jenkins_output(jenkins_url, test_name, trss_servers=["https://trss.adoptium.net"]):
+    #Get build info from TRSS
+    query_url = "https://trss.adoptium.net/api/parseJenkinsUrl"
+    query_params = {"jenkinsUrl": jenkins_url}
+    output = requests.get(query_url, query_params)
+    output_dict = json.loads(output.content)
+
+    if "output" not in output_dict:
+        raise Exception("Invalid jenkins Url")
```

Comment:
I am not sure this msg is accurate. In the case that we have output_dict, but no output, I am assuming TRSS is not responding. That means TRSS API is down or not accessible.

Reply:
I have updated the message. Please let me know if the new message looks good. Thank you!